src/lib/swapper/types.ts

```diff
@@ -159,6 +160,47 @@ export type GetUnsignedTxArgs = {
   slippageTolerancePercentageDecimal: string
 } & FromOrXpub
 
+export type EvmTradeExecutionProps = {
+  from: string
+  nonce: string
+  // TODO: required for cowswap
+  // signMessage: (messageToSign: string) => Promise<string>
+  signAndBroadcastTransaction: (transactionRequest: EvmTransactionRequest) => Promise<string>
+}
```

Comment:
no more FromOrXpub and associated logic for any of these code paths 🍻

Reply:
oh man tyvm for removing this initial band-aid